### PR TITLE
update release instruction a bit again

### PR DESCRIPTION
**Pull request by Carreau**

### CODE REVIEW

File: docs/source/development_js.rst [hunk 1]
```diff
@@ -70,10 +68,14 @@ from within JavaScript.
...
+5. You can now test the ``wheel`` and the ``sdist`` locally before uploading to PyPI.
 Make sure to use `twine <https://github.com/pypa/twine>`_ to upload the archives over SSL.
 
 .. code::
 
     $ twine upload dist/*
 
+6. If all went well, change the ``notebook/_version.py`` back adding the `.dev` suffix.
+
+7. Push directly on master, not forgetting to push `--tags`.
```
Comment (takluyver) [inline docs/source/development_js.rst]: Do you need double backticks here and on point 6? I don't think we've done the Sphinx config trick in this repo to make single backticks work like double.

Comment (Carreau) [inline docs/source/development_js.rst]: also should add to redo a `jsversion`.

**Comment by willingc** [comment]
@Carreau Added the backticks to the docs on your branch.

**Comment by Carreau** [comment]
Thanks @willingc !

**Pull request merged**